src/client.js

```diff
@@ -28,7 +28,9 @@ IrcClient.prototype._applyDefaultOptions = function(user_options) {
         nick: 'ircbot',
         username: 'ircbot',
         gecos: 'ircbot',
-        encoding: 'utf8'
+        encoding: 'utf8',
+        ping_interval: 30,
+        ping_timeout: 120,
```

Comment:
Yeah I see how it is now. Maybe bump `ping_interval` to 60 or 120 too? Should be a safer default.